lxd/container_lxc.go

```diff
@@ -477,6 +478,11 @@ type containerLXC struct {
 	storage storage
 }
 
+// OS returns the sys.OS facade instance used by this container.
+func (c *containerLXC) OS() *sys.OS {
+	return c.state.OS
+}
```

Comment:
I'm not sure what this function gains us.

    c.OS()

isn't all that much shorter than

   c.state.OS

And the latter is much clearer on what it does, as in wouldn't require me to jump to the OS() function to see what exactly that does.

Reply:
Ok, I dropped it in ac2501d ("Drop the containerLXC.OS() convenience").